src/pixelator/annotate/__init__.py

```diff
@@ -376,6 +376,8 @@ class AnnotateAnndataStatistics(typing.TypedDict):
     min_size_threshold: Optional[int]
     max_size_threshold: Optional[int]
     doublet_size_threshold: Optional[int]
+    fraction_potential_doublets: float
+    n_edges_to_split_potential_doublets: int
```

Comment:
I wonder why these are not `Optional` and e.g. the aggregates metrics are. Do you know @fbdtemme ? I guess if it is not found, will it be `None` or `0`?

Reply:
I wonder if we should set `None` here to initialize and differ from the case where we find 0 doublets.